ledger/sandbox-classic/src/main/scala/platform/sandbox/stores/ledger/inmemory/InMemoryLedger.scala

```diff
@@ -573,18 +571,22 @@ private[sandbox] final class InMemoryLedger(
 
   private def deduplicationKey(
       commandId: CommandId,
-      submitter: Ref.Party,
-  ): String = commandId.unwrap + "%" + submitter
+      submitters: List[Ref.Party],
+  ): String =
+    if (submitters.length == 1)
+      commandId.unwrap + "%" + submitters.head
+    else
+      commandId.unwrap + "%" + submitters.asInstanceOf[List[String]].sorted.distinct.mkString("%")
```

Comment:
Same as with `JdbcLedgerDao`:
```suggestion
      commandId.unwrap + "%" + submitters.sorted(Ordering.String).distinct.mkString("%")
```